common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -1493,7 +1493,8 @@ def save_problem(self, data):
             ).format(button_name=self.submit_button_name())
         return {
             'success': True,
-            'msg': msg
+            'msg': msg,
+            'html': self.get_problem_html(encapsulate=False)
```

Comment:
Note: This is used when changing problems vertically.  Should be kept in or saved answers are not persisted visually.